content/docs/guides/crosswalk/aws/vpc.md

```diff
@@ -474,6 +474,84 @@ sg.createEgressRule("outbound-access", {
 For additional details about configuring security group rules, See the
 [Security Groups for Your VPC](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html) documentation.
 
+## How to use your VPC, Security Group, and EC2 instance
+
```

Comment:
```suggestion
## How to use your VPC, Security Group, and EC2 instance

This example shows how to deploy an EC2 instance using a VPC and Security Group provisioned with the Crosswalk AWS component:

```

Reply:
Updated the content to reflect the above